mcs/class/System.Core/System.IO.Pipes/PipeStream.NotSupported.cs

```diff
@@ -0,0 +1,90 @@
+using Microsoft.Win32.SafeHandles;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.IO.Pipes
+{
+	public abstract partial class PipeStream : Stream
+	{
+		internal const bool CheckOperationsRequiresSetHandle = false;
```

Comment:
Is this used anywhere?

Reply:
Yes, from the CoreFX PAL. It is still compiled in for the .NotSupported.cs classes. I can either keep it that way or copy the whole public API here and remove the references from .sources.

Reply:
I see, let's keep this as it's